usage/README.rst

```diff
@@ -57,9 +57,16 @@ In the same directory where the data file usage.out resides, run the following i
 
 .. code-block:: bash
 
-   python usageplot.py -t w_2018_03 -d usage.out -n usage_w_2018_03
+   python usageplot.py -t w_2018_03 -d usage.out -n usage_w_2018_03 -c > nodehour.out
+
+The -t argument is the plot title (completely arbitrary), the -d argument is the name of the text file with the data (created above with usage.py), the -n argument is the name of the .png file that will be made with usageplot.py (note that you should NOT include ".png" at the end of the -n argument!!), and the -c argument will specify if you would like the plots color-coded by the Slurm jobNames that you have assigned (this will only work if singleFrameDriver.py job names start with "Co" or "Wi", mosaic.py job names start with 'mo', coaddDriver.py job names start with 'co', and multibandDriver.py job names start with 'mt').  The 'nodehour.out' file will catch the total node-hours used by slurm on the jobs whose data is being plotted.
+
+If you would prefer to not have the plots color coded, then run the follwoing in the command line:
+
+.. code-block:: bash
+
+   python usageplot.py -t w_2018_03 -d usage.out -n usage_w_2018_03 > nodehour.out
```

Comment:
I do appreciate the effort you took to properly document all options accepted by your plotting script, I really do. However, note that it is a tutorial and generally its role is to provide a prospective user something useful but not too complex to do. So in the tutorial I would limit the description to something like:

> To plot the data collected in the previous step run
```
python usageplot.py usage.out > nodehour.out
```` 
> Besides making the plot, script will also return the total node-hours used by slurm on the jobs whose data is being plotted which we caught in `nodehour.out`.

Reply:
Would you suggest that this User Guide be included in the README, just under a separate section, or would you recommend that I make a USERGUIDE.rst file for all of these details?

Reply:
That's a good question but the answer depends on the scope of your project. If you're basically done I think you could put everything in the README file.

Ideally though, especially if you continue adding new features and tools, you should create a separate subdirectory, `ldf_ops_tools/docs`, and keep the documentation for the whole package there leaving only bare minimum in the README file. A single file (describing installation, tutorials, and in-depth user guide) should be enough to start with. Then we could use [Sphinx](http://www.sphinx-doc.org/en/stable/) to create nice looking docs with minimal effort.

Reply:
Ok, I think for now I'll just settle on making a section in the README, but if I keep expanding on this code, I'll take your second suggestion above.